src/compiler/component-hydrate/bundle-hydrate-app.ts

```diff
@@ -27,7 +27,8 @@ export async function bundleHydrateApp(config: d.Config, compilerCtx: d.Compiler
         globalScriptsPlugin(config, compilerCtx),
         componentEntryPlugin(config, compilerCtx, buildCtx, build, buildCtx.entryModules),
         config.sys.rollup.plugins.nodeResolve({
-          mainFields: ['collection:main', 'jsnext:main', 'es2017', 'es2015', 'module', 'main']
+          mainFields: ['collection:main', 'jsnext:main', 'es2017', 'es2015', 'module', 'main'],
+          ...config.nodeResolve
         }),
         config.sys.rollup.plugins.emptyJsResolver(),
         config.sys.rollup.plugins.commonjs({
```

Comment:
I noticed commonjs is not reading `config.commonjs` either, could you add it too?